jukebox_radio/users/models.py

```diff
@@ -1,9 +1,23 @@
 from django.conf import settings
 from django.contrib.auth.models import AbstractUser
 from django.db import models
+from django.contrib.auth.models import User
 
 from cryptography.fernet import Fernet
 
+class UserProfile(models.Model):
+    """
+    User Profile
+    """
+    user = models.OnetoontField(User, on_delete=models.CASCADE)
+    username = models.username()
```

Comment:
We don't need username here because it's on the User model (inherits from `AbstractUser`). The username will be read from the User object, but all the other profile attributes can be read from UserProfile.

Reply:
Oh yeah you said that before. Oops! I just keep thinking that I need to add it because I don't see it.